packages/core/src/renderer/components/+namespaces/namespace-details.tsx

```diff
@@ -103,6 +104,8 @@ class NonInjectedNamespaceDetails extends React.Component<NamespaceDetailsProps
             </Link>
           ))}
         </DrawerItem>
+
+        <NamespaceTreeView root={namespace}/>
```

Comment:
It would be better if `<NamespaceTreeView namespaces={(string|object)[]} />` component would accept as input just plain array with extracted namespaces data from previous step (e.g. some extra function at api level maybe), that way you don't need to worry about apis, stores, etc. in your component tests. 
Also it's better follows single responsibility principles, UI component just displays some prepared data and don't know anything about stores, apis, etc. Wdyt?